source/native/shim.ts

```diff
@@ -0,0 +1,9 @@
+// global.process: Not the best solution, but good for now to make React Native happy.
```

Comment:
Not too bad actually. Just these stub values and then everything else has a package installed for it.